docs/deployment-key-rotation.md

```diff
@@ -3,15 +3,15 @@ Martha Deployment Key Rotation
 
 The service account key used to deploy Martha needs to be rotated every 90 days.
 
-1. Find the current BT ticket for rotating the deployment SA key. If these instructions were followed, it will be in the Slack notification that led you here. Add it to the current Jira board if it's not already there.
+1. Find the current WX ticket for rotating the deployment SA key. If these instructions were followed, it will be in the Slack notification that led you here. Add it to the current Jira board if it's not already there.
 2. [Run the full manual test on the prod environment](https://docs.google.com/document/d/1-SXw-tgt1tb3FEuNCGHWIZJ304POmfz5ragpphlq2Ng). Not strictly necessary, but this is expected to pass and sets a current baseline to avoid surprises if the post-rotation test deploy fails.
 3. Using your `@firecloud.org` account, [create a new key for the `cloud-functions-account@broad-dsde-prod.iam.gserviceaccount.com` service account](https://console.cloud.google.com/iam-admin/serviceaccounts/details/107440104000315564432/keys?project=broad-dsde-prod).
    1. Click `[ADD KEY]` > `Create new key`
    2. Select the JSON key type
    3. Click `[CREATE]` and make sure the key `.json` file downloads to your computer
 4. Write this new key to vault.
 ```
-vault write secret/dsde/martha/prod/deploy-account.json @[key .json file]
+vault write secret/dsde/martha/prod/deploy-account.json @/path/to/key.json
```

Comment:
I thought the `[]` were part of the syntax and had to check the docs to figure it out